libs/features/personalization/personalization.js

```diff
@@ -289,6 +293,14 @@ function getSection(rootEl, idx) {
     : rootEl.querySelector(`:scope > div:nth-child(${idx})`);
 }
 
+function registerCustomAction(cmd) {
```

Comment:
should we differentiate normal MEP rules from these ones?
if we expose all the MEP rules and have a flag when they are successfully executed at page load, then blocks such as `merch-card-collection` can go over the rules without that flag and find matching rules and execute them.

Reply:
they are, all normal processing is stopped if identified action is custom, and then put in config.mep.custom

Reply:
actually I meant, do we need to differentiate, sorry for the language.